### DISCUSSION TITLE
更新后502了什么原因。

### DISCUSSION THREAD

**Discussion by zaizaiyou** [AlistGo/alist#480] [Q&A] [answered]
使用curl -fsSL "https://nn.ci/alist.sh" | bash -s update   2.0.7更新2.0.9然后502了

**Comment by xhofe** [comment, marked as answer]
配置文件：config.json
assets的值从zhimg改成空重启即可。即：
"assets":"",
同 #455

**Reply by zaizaiyou** [reply]
github是你第二个家嘛，每次都秒回复。

**Reply by xhofe** [reply]
有邮件通知啊。。。

**Reply by zaizaiyou** [reply]
打扰了打扰了
